Source/WebCore/page/scrolling/mac/ScrollerMac.mm

```diff
@@ -369,6 +370,8 @@ - (void)invalidate
 
 FloatPoint ScrollerMac::convertFromContent(const FloatPoint& point) const
 {
+    if (m_pair.mouseInContentArea())
+        return m_lastKnownMousePosition;
```

Comment:
This doesn’t make sense. The function is asked to convert `point` into the scrollbar’s coordinate system. It’s not right to just return `m_lastKnownMousePosition` here.

Reply:
I think the confusion here is ScrollerPairMac has m_lastKnownMousePosition which is the raw mouse position, while each ScrollerMac has m_lastKnownMousePosition which is the raw mouse position relative to that scroller. I renamed each of these to be more clear. ScrollerMac::convertFromContent is called with the raw mouse position so if m_pair.mouseInContentArea() is true we know the converted mouse position is updated by the Web Process.